.github/workflows/deploy-mainnet.yml

```diff
@@ -1,12 +1,16 @@
 name: Deploy to Mainnet network
 
-# Run on pushes to master or PRs
+# Run on pushes to main or PRs
 on:
   # Pull request hook without any config. Launches for every pull request
   pull_request:
+    branches:
```

Comment:
The branch indicated here is the destination branch. So this will run on PRs which use `main`, `release/**` and `mainnet/**`as base branch

To indicate when adding changes to a branch that fulfills a pattern name should be done in `push` hook. That's why we didn't add something like this yet

Reply:
Oops.
Then I suggest we keep only `main` there.